src/libos/src/fs/syscalls.rs

```diff
@@ -627,9 +627,6 @@ pub async fn do_sendfile(
         from_user::check_mut_ptr(offset_ptr)?;
         Some(unsafe { offset_ptr.read() })
     };
-    if count < 0 {
```

Comment:
cannot just delete the check. According to  the manual, the `count` argument can be a negative number,  you can change the type of `count` from `size_t` to `isize`

Reply:
I saw a conflict about this.
In Linux manual, the count is defined as size_t which is unsigned type.
Also in th man page, there is a description "set errno EINVAL  if count is negative."

So I checked it in the Linux kernel, there is no such negative check.
https://elixir.bootlin.com/linux/latest/source/fs/read_write.c#L1175

Reply:
Linux dose check the value:
```
int rw_verify_area(int read_write, struct file *file, const loff_t *ppos, size_t count)
{
	if (unlikely((ssize_t) count < 0))
		return -EINVAL;

```

Reply:
@liqinggd Good catch! I will add the check back.